src/constants.py

```diff
@@ -15,6 +17,16 @@
     "ContentDate:End": "content_date_end",
     "Checksum:Algorithm": "checksum_algorithm",
     "Checksum:Value": "checksum_value",
+    "IsSealed": "sealed_state",
```

Comment:
This is for POST requests, creating a single catalog item in the table. Is it required?  @ub0005 Can you please double-check with @DImuthuUpe

Reply:
we are using CATALOGUE_CSV_COLUMN_MAPPER in bulk upload @NISH1001 
Do you want to check whether IsSealed is needed as a mandatory value to send?